resources/measures/upgrade_df_thermostat_control_load_shift/measure.rb

```diff
@@ -461,12 +482,15 @@ def applicable_thermostats(model)
     puts("### Applicability check...")
     applicable_building_types = [
       # "Hotel",
-      # "SmallOffice",
-      # "MediumOffice",
+      "SmallOffice",
+      "MediumOffice",
       "LargeOffice",
       "OfL",
       "OfS",
-      "Office"
+      "Office",
+      "Warehouse",
+      "PrimarySchool",
```

Comment:
are there space types in schools that you are excluding? Is it reasonable to apply these to gyms, cafeteria, etc.?

Reply:
We assume a uniform applicability to all space type to simplify the measure (for now) with the following considerations:
1. The control is applicable to the majority of spaces in the building;
2. The overestimated potential of applying the measure would compensate the underestimated potentia - for example, the lights in corridor area could be dimmed more while the lights in gym should be dimmed less (or skipped) - and such trade-offs are assumed to have negligible impact on the overall performance (especially in stock level).

These are explained in the documentation as well. Let me know your opinion on this.